docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -0,0 +1,69 @@
+// tag::ess[]
+
+// QUESTION: Should we add a warning here about updating to 7.13 and include a
+// link to the upgrade docs? 
+
+{ecloud} runs a hosted version of {fleet-server}. To use {fleet-server}, start
+the service:
+
+//TODO: Describe how to start the service.
+
+When you're done, log in to {kib} and go to *Management > {fleet}*. Click the
+*Agents* tab to confirm that {fleet-server} is running and healthy.
+
+//TODO: Add screen capture.
+
+On the *Policies* tab, notice that the {agent} is enrolled in the Elastic Cloud
+policy. This policy is managed by {ecloud}. You cannot modify it.
+
+// end::ess[]
+
+// tag::self-managed[]
+
+To deploy {fleet-server} on a self-managed cluster, install an {agent} and
+enroll it in the default {fleet-server} policy. The policy is created
+automatically when you run {fleet} for the first time.
+
+To add {fleet-server}:
+
+//QUESTION: Do we want to show how to call the API to generate the token, or
```

Comment:
Focus on easy setup. Ideally the setup would have enough docs directly in the UI.